api/v2.0/swagger.yaml

```diff
@@ -1919,3 +1949,12 @@ definitions:
         type: boolean
       default:
         type: boolean
+  Icon:
```

Comment:
What's the reason you have to put it in a JSON?
Why not put the content-type in the header and return the content in response body directly?

Reply:
It's requested by the front end guy. 
BTW, the content is based64 encoded, so if we put the content in the response body directly, the content-type header will not be image/xxx either.